### PR TITLE
Added the Virtual Keyboard page

### PR DESCRIPTION
**Pull request by ArturA-MS**

### CODE REVIEW

File: BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    <TextBlock Text=" " Width="20" />
+                    <ToggleSwitch x:Name="VirtualKeyboardSwitch"
+                                  Header="Discoverable and Connectable"
+                                  IsOn="{x:Bind ViewModel.IsKeyboardEnabled, Mode=TwoWay}"
+                                  IsEnabled="{x:Bind ViewModel.IsSettingUp.Equals(x:False), Mode=OneWay}"
+                                  IsFocusEngagementEnabled="False"
+                                  AllowFocusOnInteraction="False"
+                                  AllowFocusWhenDisabled="False"
+                                  IsTabStop="False" />
+                </StackPanel>
```
Comment (FrankGorgenyi) [inline BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml:119]: Show a connected devices list please.

File: BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                <TextBlock Visibility="{Binding IsOn, ElementName=VirtualKeyboardSwitch, Converter={StaticResource VisibleWhenTrueConverter}}"
+                           Text="While this application is in focus, keystrokes from your physical keyboard will be routed to any and all paired clients that are connected and subscribed to the HID notifications."
+                           TextWrapping="WrapWholeWords" />
+            </StackPanel>
```
Comment (FrankGorgenyi) [inline BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml:158]: Add a note explaining the Android bug, specifically when they need to toggle the radio. If we have a public bug link they can upvote consider placing it here. :)

**Review by FrankGorgenyi — CHANGES_REQUESTED** [review]
Great work on short notice thank-you. 👍

File: BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml (hunk 1 above)
Comment (ArturA-MS) [reply BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml:119]: Done!

File: BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml (hunk 2 above)
Comment (ArturA-MS) [reply BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml:158]: Let me know if you think the wording now is sufficient.

**Review by FrankGorgenyi — APPROVED** [review]

**Pull request merged**